corehq/apps/users/management/commands/make_emails_lowercase.py

```diff
@@ -0,0 +1,14 @@
+from django.core.management import BaseCommand
+
+from corehq.apps.users.models import CouchUser
+
+
+class Command(BaseCommand):
+    help = "Makes emails into lowercase"
+
+    def handle(self, *args, **options):
+        for couch_user in CouchUser.all():
+            if couch_user.email and any(char.isupper() for char in couch_user.email):
+                print couch_user.email
+                couch_user.email = couch_user.email.lower()
+                couch_user.save()
```

Comment:
they have not always been lowercase, but I did not add transforming usernames to this command because you have to be more careful when lowercasing usernames, since you don't want to create users with duplicate usernames (ex: distinct users eSoergel and Esoergel).  one could add functionality to check this here, if desired.  However this shouldn't be an issue going forward on prod, staging, and india.